CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
+## [3.1.0] - 06.02.2024
```

Comment:
Because this is "only" a bug fix shouldn't only the version increase to `3.0.18`? A minor version increase would mean the we shipped a new feature that is non-breaking.

Reply:
I already had this in my mind in the Daily. This needs to be a major Version. 4.0
As its no longer compatible with all the previous versions.